test/test_kubeclient.rb

```diff
@@ -4,6 +4,10 @@ def open_test_json_file(name)
   File.new(File.join(File.dirname(__FILE__), 'json', name))
 end
 
+def open_test_text_file(name)
```

Comment:
can it be unified with the open_test_json_file? a method can receive both the name and the needed extension. it will reduce code duplication.